styles.css

```diff
@@ -346,3 +356,96 @@
 .footer-heading span {
   margin-left: 12px;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-form h2 {
+  font-family: 'Inter', sans-serif;
+  margin: 69px 24px 0;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.input-font {
+  text-align: left;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  width: 295px;
+  height: 18px;
+  border: none;
+  padding: 0;
+}
+
+.container {
+  border-bottom: 1px solid #dbd8d7;
+  display: flex;
+  justify-content: center;
+  padding: 15px 16px;
+  margin-left: 23px;
+  margin-right: 25px;
+}
+
+.last-div {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+}
+
+#sms {
+  width: 303px;
+  height: 90px;
+  background-color: rgb(251, 248, 247);
+  padding: 12px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+#submit {
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  padding: 12px;
+  margin: 0 24px;
+  margin-top: 24px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  margin-bottom: 78px;
+}
+
+#submit:link {
+  background-color: #f55800;
+  color: #fff;
+}
+
+#submit:visited {
+  background-color: #ff6b00;
+}
+
+#submit:hover {
+  background-color: #dbd8d7;
+  color: #e05304;
+}
+
+#submit:active {
+  background-color: #e05304;
+  color: #fff;
+}
```

Comment:
Good job implementing the button interactions 🥇 However, please make sure to follow the Figma design guidelines while adding interactions, as suggested by the reviewer [here](https://github.com/tumainimaganiko/Portfolio/pull/3#discussion_r1151001516) ⚒
- [ ] I noticed that you are using  `background-color: #dbd8d7;` whereas in Figma template it is `#FFF5E1`. Also please make sure to implement the border for `hover` interaction. See images below for more clarity 👇
- [ ]  Similarly, please fix the color. In Figma template it is `#F55800` for `hover` and `#F4F1F0` for `active`

Your design | Figma template
---| ---|
![image](https://user-images.githubusercontent.com/106119105/228358824-29e6b53e-6063-4398-84ff-ba128170fb06.png) | ![image](https://user-images.githubusercontent.com/106119105/228358726-5aaee786-1aa5-46e5-9ad2-c1a443358f78.png)